### DISCUSSION TITLE
2022 database engines ranking

### DISCUSSION THREAD

**Discussion by dev-heeseok** [dev-heeseok/web-react#17] [Announcements]
@kimkmin357 @DeukHan 
[Database Ranking](https://db-engines.com/en/ranking) 을 확인할 수 있는 웹 사이트 입니다.
2022년 MongoDB 는  5번째로 많이 쓰는 database 라고 하네요.

**Comment by kimkmin357** [comment]
> @kimkmin357 @DeukHan [Database Ranking](https://db-engines.com/en/ranking) 을 확인할 수 있는 웹 사이트 입니다. 2022년 MongoDB 는 5번째로 많이 쓰는 database 라고 하네요.

좋은정보 감사합니다 ^^

**Comment by DeukHan** [comment]
오라클이 1등이네요 
<img width="1022" alt="image" src="https://user-images.githubusercontent.com/104338516/169295915-3561bfc9-1267-4e8f-a88e-1a6a506cb9d3.png">

**Reply by dev-heeseok** [reply]
참고로 오라클은 비용이 비싸서 대기업에서 많이 쓴다고 하더라구요. 그래서 독보적인 일등이라고 합니다.
